docs/examples/Example Machine Learning - OQMD.ipynb

```diff
@@ -102,7 +106,11 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
+<<<<<<< HEAD
```

Comment:
@ncsa-mo Why are these here?
I'm having trouble understanding the cause for changes to this notebook.